pokatlas.py

```diff
@@ -42,6 +47,10 @@ def get_atlas(path: pathlib.Path) -> Atlas:
 
     return atlas
 
+def get_image_hash(image_path: pathlib.Path) -> str:
+    with open(image_path, "rb") as f:
+        return hashlib.md5(f.read()).hexdigest()
```

Comment:
I would prefer not using `with open`, simpler to just read with the pathlib path.
`return hashlib.md5(image_path.read_bytes()).hexdigest()`

Reply:
Great point, have fixed this thank you :)